projects/fusion-ui/components/chip-filter/common/base/chip-filter-component-configurations.ts

```diff
@@ -1,16 +1,16 @@
 export interface ChipFilterComponentConfigurations {
     id?: number | string;
-    close?: boolean;
     disabled?: boolean;
-    selected?: boolean;
-    type?: ChipType;
+    type?: ChipFilterType;
+    close?: boolean;
 }
 
 export enum ChipTypeToClass {
-    DateFilter = 'fu-date-filter',
+    UnRemoveAbleSelect = 'fu-un-removable-filter',
     AddFilter = 'fu-add-filter',
-    ChipSelect = 'fu-chip-select',
-    ChipDropdown = 'fu-chip-dropdown'
+    RemoveAbleSelect = 'fu-removable-filter',
+    ChipFilter = 'fu-chip-filter'
 }
 
-export type ChipType = 'DateFilter' | 'AddFilter' | 'ChipSelect' | 'ChipDropdown';
+export type ChipType = 'UnRemoveAbleSelect' | 'AddFilter' | 'RemoveAbleSelect' | 'ChipFilter';
```

Comment:
I needed to declare types to be able to order the chips in the ChipFilters
because, chips selected without a close button need to be placed to the left(in ascending order by id), after that the chips selected with a close button(in ascending order by id), and then chip filters and addFilter